assets/dev/js/editor/components/ai/page-content.js

```diff
@@ -0,0 +1,89 @@
+import { useEffect } from 'react';
+import FormText from './pages/form-text';
+import Connect from './pages/connect';
+import FormCode from './pages/form-code';
+import GetStarted from './pages/get-started';
+import Loader from './components/loader';
+import useUserInfo from './hooks/use-user-info';
+import WizardDialog from './components/wizard-dialog';
+import PromptDialog from './components/prompt-dialog';
+import UpgradeChip from './components/upgrade-chip';
+
+const PageContent = (
+	{
+		type,
+		controlType,
+		onClose,
+		getControlValue,
+		setControlValue,
+		additionalOptions,
+	} ) => {
+	const { isLoading, isConnected, isGetStarted, connectUrl, fetchData, hasSubscription, credits } = useUserInfo();
+
+	useEffect( () => {
+		fetchData();
+	}, [] );
```

Comment:
feels weird that it's here instead of the custom hook

Reply:
Moved.